content/organization/tech/email/index.en.md

```diff
@@ -6,146 +6,55 @@ weight: 25
 
-1. Go to your Microsoft account (e.g.: hotmail, msn, outlook,etc), then click on the **Settings** (the gear)/**Configuración** (el engranaje) icon on the top-right corner and select **Connected accounts/Cuentas conectadas**
-
-![Setting -> Connected accounts](outlook1.png)
-
-2. Choose **Other email accounts/Otras cuentas de correo electrónico**.
-
-![Another mail account](outllok2.png)
-
-3. On the _Connect your email account_ page, enter your full name, email address and password as shown in the screenshot below. Also select the option **Manual configuration/Configurar manualmente las opciones de la cuenta (cuenta POP, IMAP o Solo envío)** and then click **OK/Aceptar**
-
-![Full name, emal address ans password](outlook3.png)
-
-4. Choose **Setting IMAP or SMTP connection/Configuración de conexión de IMAP o SMTP** and click **OK/Aceptar**
-
-![Select IMAP or SMTP](outlook4.png)
-
-5. At the **New IMAP account connection/Nueva conexión de cuenta IMAP** page, enter the email address of the account you're connecting in the email address box. In the user name box, enter the email address again, and then enter your email password in the Password box as shown in the screenshot below.
-
-![Account information](outlook5.png)
-
-6. In the **Server Information/Información del Servidor** section use these data
-
-    - Incoming IMAP server/Servidor entrante (IMAP):  mail.rladies.org
-    
-    - Incoming server port/Puerto de servidor entrante: 143
-
-    - Authentication/Autenticación: Basic/Básica
-
-    - Encryption/Cifrado: TLS
-
-and click **OK/Aceptar**
-
-![Outlook menu for 'Connect your email account'](outlook6.png)
-
-7. Outlook shows a message telling you that the account was successfully connected. Click **OK/Aceptar**. You see the account as in the screenshot below.
-
-![Account conected](outlook7.png)
-
-### Mac Mail
-
-1. Click on Mail > Add Account ...
-
-2. Choose "Other Mail Account"
-
-![Mac Mail menu for choosing what type of accounts to add](macmail1.png)
-
-3. Enter email address and password.
+### Smartphone
 
-![Mac Mail menu for entering email address and password](macmail2.png)
+Here are instructions for adding an email account to an [iPhone](https://support.google.com/mail/topic/2467017?hl=en&ref_topic=2451730&sjid=12348347883239577326-SA) and [Android](https://support.google.com/mail/topic/2451697?hl=en&ref_topic=2451730&sjid=12348347883239577326-SA).
 
-4. Choose POP and enter "mail.rladies.org" for both servers. Click "Sign In"
+### Webmail
 
-![Mac Mail menu for server settings](macmail3.png)
+1. On your computer, go to [Gmail](https://mail.google.com/).
+2. Enter your Google Account email or phone number and password.
+	2.1. If information is already filled in and you have to sign in to a different account, click _Use another account_.
+    2.2. If you get a page that describes Gmail instead of the sign-in page, at the top right of the page, click [Sign in](https://accounts.google.com/ServiceLogin?service=mail).
```

Comment:
```suggestion

	2.2. If you get a page that describes Gmail instead of the sign-in page, at the top right of the page, click [Sign in](https://accounts.google.com/ServiceLogin?service=mail).
```